### ISSUE TITLE
Strawberry generates weird copies/hardlinks of 'strawberry.conf' file

### ISSUE THREAD

**Issue by kanunnikau** [strawberrymusicplayer/strawberry#1175]
- [🗸] I have checked the [FAQ](https://wiki.strawberrymusicplayer.org/wiki/FAQ) for answers.
- [🗸] I have checked the [Changelog](https://github.com/strawberrymusicplayer/strawberry/blob/master/Changelog) that the issue is not already fixed. 
- [🗸] I believe this issue is a bug, and not a general technical issue, question or feature requests that can be discussed on the [forum](https://forum.strawberrymusicplayer.org/). 

**Describe the bug**
While trying to figure out why are all equalizer presets zeroed and therefore not working (still haven't solved it, but found a workaround, so it's good enough for now) I came across a puzzle.
Strawberry seems to generate a ton of some kind of links of the 'strawberry.conf' file every time it is launched and exited.

**To Reproduce**
- Check '~/.config/strawberry/' directory. There should be only one 'strawberry.conf' file.
- Launch Strawberry.
- See a ton of 'strawberry.conf' entries.
- Quit Strawberry.
- See a ton *more* of 'strawberry.conf' entries.
- Launch Strawberry.
- See a ton *more* of 'strawberry.conf' entries.
- Quit Strawberry.
- See a ton *more* of 'strawberry.conf' entries.

It just seems to create more and more of those entries. Removing *one* of them removes all.
It seems that the last one is the "real" file — if I try to remove any other than the last they will be all gone, but throw an error complaining that such file cannot be found. If I remove the last one they all disappear together, quietly.

**Expected behavior**
In the end, there can be only one ('strawberry.conf' config file).

**Screenshots:**
Please see screenshots of the terminal window (output of 'ls' and 'ls -la'): 
![2023-04-07_13-04](https://user-images.githubusercontent.com/2588408/230599360-cca87618-53e9-4137-af13-19adf7ae0158.png)
![2023-04-07_13-05](https://user-images.githubusercontent.com/2588408/230599365-fa09b093-3a88-4017-b69c-166c7c0ba92e.png)

And video recording of:
- launching Strawberry,
- seeing extra entries,
- quitting Strawberry,
- // repeating those steps several times
- removing *one* entry and seeing them all disappear.
[output.webm](https://user-images.githubusercontent.com/2588408/230600535-78ad5c5a-c37a-445c-bb84-ce8ffd2052ba.webm)

**System Information:**
 - Operating system: openSuse Tumbleweed 20230405
 - Strawberry Version: 1.0.17

**Comment by kanunnikau** [comment]
This was getting weird, so I've reinstalled Strawberry, removed and recreated the folder, let Strawberry recreate the config file — and everything is fine now.
I guess the issue can be closed.
Probably it was something caused by the OS/filesystem as even the backed up files (copied to a different folder) were removed when I deleted the strawberry folder.
Very weird but unlikely to be caused by Strawberry itself :)
